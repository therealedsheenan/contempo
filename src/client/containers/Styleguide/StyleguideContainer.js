import React from 'react'

const StyleguideContainer = () => {
  return (
    <div className='Styleguide' role='styleguide'>
      <p>this is the styleguide</p>
    </div>
  )
}

export default StyleguideContainer
