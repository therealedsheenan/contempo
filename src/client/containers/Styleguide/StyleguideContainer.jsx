import React from 'react';

const StyleguideContainer = () => (
  <div className="Styleguide" role="styleguide">
    <p>this is the styleguide</p>
  </div>
  );

export default StyleguideContainer;
