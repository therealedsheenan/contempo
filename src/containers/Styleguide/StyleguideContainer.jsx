// @flow

import React from 'react';

const StyleguideContainer = () => (
  <div className="Styleguide">
    <p style={{ textAlign: 'center' }}>this is the style guide!</p>
  </div>
);

export default StyleguideContainer;
