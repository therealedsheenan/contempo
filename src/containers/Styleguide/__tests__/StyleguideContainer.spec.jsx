// @flow

import React from 'react';
import renderer from 'react-test-renderer';

import StyleguideContainer from '../StyleguideContainer';

test('Home existence', () => {
  const component = renderer.create(<StyleguideContainer />).toJSON();

  // correct home component
  expect(component).toMatchSnapshot();
});
