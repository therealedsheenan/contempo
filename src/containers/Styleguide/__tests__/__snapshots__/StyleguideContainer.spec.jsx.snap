// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home existence 1`] = `
<div
  className="Styleguide"
>
  <p
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    this is the style guide!
  </p>
</div>
`;
